res/values/strings.xml

```diff
@@ -462,16 +462,16 @@
     <string name="pref_watch_mvbox_folder">Watch DeltaChat folder</string>
     <string name="pref_send_copy_to_self">Send copy to self</string>
     <string name="pref_auto_folder_moves">Automatic moves to DeltaChat folder</string>
-    <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox folder</string>
+    <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the \"Inbox\" folder</string>
```

Comment:
Not sure why "folder" adds anything here.

Reply:
yip, maybe "...avoid cluttering the inbox" is just fine.